src/Security/Authentication/test/JwtBearerTests.cs

```diff
@@ -1162,6 +1202,17 @@ private static async Task<IHost> CreateHost(Action<JwtBearerOptions> options = n
                                 // Simulate Forbidden
                                 await context.ForbidAsync(JwtBearerDefaults.AuthenticationScheme);
                             }
+                            else if (context.Request.Path.StartsWithSegments("/forbiddenByMultipleAuthSchemas"))
+                            {
+                                // Simulate Forbidden By Multiple Authentication Schemas
+                                var schemaCountFromPath = context.Request.Path.ToString().Split('/').Last();
+                                Int32.TryParse(schemaCountFromPath, out int schemaCount);
+
+                                for( int i = 0; i <= schemaCountFromPath.Length; i++)
+                                {
+                                    await context.ForbidAsync(JwtBearerDefaults.AuthenticationScheme);
```

Comment:
Calling Forbid on the same scheme multiple times isn't really the same as calling Forbid once on multiple schemes.

Reply:
Yes, that is true, however it was still producing the same error for unit test simulation purposes.  However, I made some modifications to this unit test, in the hopes that it more closely resembles the multiple authentication schema rejection response as forbidden scenario.  Please let me know if this change isn't sufficient and if you have a preference for how this simulated scenario should look in a unit test.  Thanks so much @Tratcher